x/ethbridge/keeper/nativeTokenReceiverAccount.go

```diff
@@ -0,0 +1,49 @@
+package keeper
+
+import (
+	"bytes"
+
+	"github.com/Sifchain/sifnode/x/ethbridge/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	protobuftypes "github.com/gogo/protobuf/types"
+)
+
+func (k Keeper) SetNativeTokenReceiverAccount(ctx sdk.Context, nativeTokenReceiverAccount sdk.AccAddress) {
```

Comment:
Update naming to `SetCrossChainFeeReceiverAccount`

Reply:
Rename source files as well

Reply:
renamed